### ISSUE TITLE
what is .npy data?

### ISSUE THREAD

**Issue by IG71Y** [khdlr/SoundingEarth#1]
Data in. npy format appears in the advance.py file under downstream. But I can‘t find it,So can you provide it? Thanks you:)

**Comment by khdlr** [comment]
Right, that was an oversight on my end. The `.npy` files are pre-computed spectrogram data. I've uploaded them for you here: https://we.tl/t-seLLAyDzAX

**Comment by sun-0704** [comment]
Sorry, https://we.tl/t-seLLAyDzAX appearance that this transfer has expired and is not available anymore. May I ask how to obtain this file now？ Thank you.

**Comment by khdlr** [comment]
I have re-uploaded the spectrogams as a Github release, you can download them here:
https://github.com/khdlr/SoundingEarth/releases/download/spectrograms/advance_spectrograms.tar.gz
